nominatim/cli.py

```diff
@@ -61,7 +61,10 @@ def __init__(self, prog, description):
     def nominatim_version_text():
         """ Program name and version number as string
         """
-        return "Nominatim version %s.%s.%s.%s\n" % version.NOMINATIM_VERSION
+        text = 'Nominaim version %s.%s.%s.%s' % version.NOMINATIM_VERSION
```

Comment:
Good thing we have tests. They caught that typo in Nominatim.

Reply:
Funny. I found some more even, will submit in a separate PR.